src/PersonalityQuiz.tsx

```diff
@@ -0,0 +1,381 @@
+                    `}
+                >
+                    {number + '.'}
+                </span>
+                {text}
+            </legend>
+        </div>
+        {imageUrl && (
+            <img
+                className={css`
+                    width: 100%;
+                `}
+                src={imageUrl}
+            />
+        )}
+        <div
+            className={cx(
+                radioButtonWrapperStyles,
+                css`
+                    label {
+                        :hover {
+                            background-color: ${hasSubmittedAnswers
+                                ? neutral[97]
+                                : neutral[86]};
+                        }
+                        /* TODO: apply same styles on focus (requires source update) */
+                    }
+                `,
+            )}
+        >
+            <RadioGroup name={questionId}>
+                {answers.map((answer) => (
+                    <Radio
+                        key={answer.id}
+                        value={answer.text}
+                        label={answer.text}
+                        data-testid={answer.id}
+                        data-answertype={
+                            selectedAnswer === answer.id
+                                ? 'selected-enabled-answer'
+                                : 'unselected-enabled-answer'
+                        }
+                        disabled={hasSubmittedAnswers}
+                        onChange={() => updateSelectedAnswer(answer.id)}
+                        checked={selectedAnswer === answer.id}
+                    />
+                ))}
+            </RadioGroup>
+        </div>
+    </fieldset>
+);
+
+const missingAnswersStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+    padding-bottom: ${space[3]}px;
+    color: ${text.error};
+`;
+
+export const MissingAnswers = (): JSX.Element => (
+    <div className={missingAnswersStyles} data-testid="missing-answers">
+        You have not answered all the questions.
+    </div>
+);
+
+const resultWrapperStyles = css`
+    background-color: ${neutral[93]};
+    margin-top: ${space[3]}px;
+    margin-bottom: ${space[3]}px;
+    padding: ${space[2]}px;
+`;
+const resultHeaderStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+    color: ${neutral[20]};
+    padding-bottom: ${space[1]}px;
+`;
+
+const resultDescriptionStyles = css`
+    ${textSans.medium()}
+    color: ${neutral[46]};
+`;
+
+export const Result = ({
+    resultBuckets,
+    sharingIcons,
+}: {
+    resultBuckets: ResultsBucket;
+    sharingIcons?: JSX.Element;
+}): JSX.Element => (
+    <div className={resultWrapperStyles}>
+        <div className={resultHeaderStyles}>{resultBuckets.title}</div>
+        <div className={resultDescriptionStyles}>
+            {resultBuckets.description}
+        </div>
+        {sharingIcons && (
+            <Fragment>
+                <hr />
```

Comment:
I could just lift this entire wrapper? that way we could have `DividerBlockComponent` from DCR?

Reply:
I think you can just replace `<hr />` with `<DividerBlockComponent />` - it's a styling spacing thing I guess

Reply:
`DividerBlockComponent` doesnt exist in atoms rendering, it only exists in DCR

Reply:
@SiAdcock !

Reply:
So there's a campaign to add support for a native Composer divider (instead of the * * * business) and arguably, even though this is a very simple thing, it would be nice to encapsulate it into a component in Source. The future could involve text in dividers, colours, weights, etc.